model/histogram/float_histogram.go

```diff
@@ -1033,3 +1079,115 @@ func mergeToSchema(originSpans []Span, originBuckets []float64, originSchema, ta
 
 	return targetSpans, targetBuckets
 }
+
+func mergeTwoSpans(schema int32, threshold float64, spansA []Span, bucketsA []float64, spansB []Span, bucketsB []float64) ([]Span, []float64) {
```

Comment:
I think the name is misleading. This is not really merging spans, it's mostly adding buckets. How about calling it `addBuckets` (similar to the previous `addBucket`)?

Reply:
That makes sences, since the logic is also similar to `addBucket`